pages/web/domains/dns_dnssec/guide.es-es.md

```diff
@@ -1,64 +1,87 @@
+
+**Descubra cómo activar DNSSEC para su dominio para protegerlo contra el "cache poisoning".**
+
+Para más información sobre el funcionamiento del **DNSSEC**, consulte nuestra página "[Comprender DNSSEC](https://www.ovhcloud.com/es-es/domains/dnssec/){.external}".
+
+Si desea más información sobre estos temas, no dude en consultar nuestras guías sobre [los servidores DNS OVHcloud](/pages/web/domains/dns_server_general_information/) y sobre la [edición de una zona DNS OVHcloud](/pages/web/domains/dns_zone_edit/).
 
 ## Requisitos
 
-- Tener un dominio registrado en OVHcloud.
+- Tener un dominio registrado con OVHcloud.
 - El dominio debe tener una extensión compatible con DNSSEC.
-- Estar conectado al [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es){.external}, en la sección `Web Cloud`{.action}.
+- Estar conectado a su [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es){.external}, parte `Web Cloud`{.action}.
 
 ## Procedimiento
 
-Existen dos formas de activar DNSSEC:
+Existen dos formas de activar el **DNSSEC**:
 
-- **Si el dominio utiliza los servidores DNS de OVHcloud**, puede realizar la activación directamente desde el área de cliente.
+- **Su dominio utiliza los servidores DNS de OVHcloud** : la activación se realiza en un clic desde su [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es){.external} (si esta no está activada por defecto).
 
-- **Si su dominio no utiliza los servidores DNS de OVHcloud**, debe ponerse en contacto con el proveedor que gestiona la configuración DNS y solicitarle todos los parámetros. A continuación, acceda a la sección `Web Cloud`{.action}. Haga clic en `Dominios`{.action} y, a continuación, seleccione el dominio correspondiente en la lista.
-Haga clic en la pestaña `Registros DS`{.action}, a continuación, haga clic en el botón Editar de la derecha y, por último, en el botón `+`{.action}.
-Rellene los 4 campos "Key tag", "Flag", "Algorithm", "Public key (codificado en base 64)" con los datos que le ofrece su actual proveedor.
+- **Su dominio no utiliza los servidores DNS de OVHcloud** : Contacte con el proveedor que gestione la configuración DNS del dominio para pedirle su configuración. Conéctese a su [área de cliente de OVHcloud](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.es/&ovhSubsidiary=es){.external} y acceda a la sección `Web Cloud`{.action}. En la columna izquierda, haga clic en `Dominios`{.action} y seleccione el dominio correspondiente en la lista.</br>
+Seleccione la pestaña `Registros DS`{.action}, haga clic en el botón `Editar`{.action} a la derecha y, seguidamente, en el botón `+`{.action} que aparece.</br>.
+Ya puede introducir los 4 campos "Key Tag", "Flag", "Algoritmo", "Clave pública (codificada en base 64)" con los datos comunicados por su actual proveedor.
```

Comment:
Are the 4 fields translated in the manager?

Reply:
Yes, Itranslated with a manager v6 opened